src/screens/Dashboard/screens/Login/styles.module.scss

```
@@ -0,0 +1,82 @@
+
+.app {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+}
+
+.head {
+  height: 8px;
+  width: full;
+  background-color: #00ADEE;
+}
+.rectangle {
+    height: 634px;
+    width: 300px;
+    background-color: #F4F4F4;
+}
+.rectangle-1 {
+  height: 32px;
+  width: 252px;
+  border-radius: 5px;
+  background-color: #FFF;
```

- add color in global css file _colors.scss
- write here variable colors using name convention in https://chir.ag/projects/name-that-color/

```suggestion
  background-color: $wildSand;
```